stm32 usb compile failure

When compiling the STM32 USB driver, I get a compile error:

```
CC:  chip/stm32_otghsdev.c chip/stm32_otghsdev.c: In function 'stm32_ep_allocbuffer':
chip/stm32_otghsdev.c:4367:33: error: 'privep' undeclared (first use in this function); did you mean 'pipe2'?
   usbtrace(TRACE_EPALLOCBUFFER, privep->epphy);
                                 ^~~~~~
                                 pipe2
chip/stm32_otghsdev.c:4367:33: note: each undeclared identifier is reported only once for each function it appears in
chip/stm32_otghsdev.c: In function 'stm32_ep_freebuffer':
chip/stm32_otghsdev.c:4388:32: error: 'privep' undeclared (first use in this function); did you mean 'pipe2'?
   usbtrace(TRACE_EPFREEBUFFER, privep->epphy);
                                ^~~~~~
                                pipe2
```
code: https://github.com/apache/nuttx/blob/master/arch/arm/src/stm32/stm32_otghsdev.c#L4367

Changing `privep` to `ep` does not fix the compile error.  I'm not sure what this is supposed to be in the first place.

--- a/arch/arm/src/stm32/stm32_otghsdev.c
+++ b/arch/arm/src/stm32/stm32_otghsdev.c
@@ -4364,7 +4364,7 @@ static void stm32_ep_freereq(struct usbdev_ep_s *ep,
 #ifdef CONFIG_USBDEV_DMA
 static void *stm32_ep_allocbuffer(struct usbdev_ep_s *ep, uint16_t bytes)
 {
-  usbtrace(TRACE_EPALLOCBUFFER, privep->epphy);
+  usbtrace(TRACE_EPALLOCBUFFER, ((struct stm32_ep_s *)ep)->epphy);
 
 #ifdef CONFIG_USBDEV_DMAMEMORY
   return usbdev_dma_alloc(bytes);
@@ -4385,7 +4385,7 @@ static void *stm32_ep_allocbuffer(struct usbdev_ep_s *ep, uint16_t bytes)
 #ifdef CONFIG_USBDEV_DMA
 static void stm32_ep_freebuffer(struct usbdev_ep_s *ep, void *buf)
 {
-  usbtrace(TRACE_EPFREEBUFFER, privep->epphy);
+  usbtrace(TRACE_EPFREEBUFFER, ((struct stm32_ep_s *)ep)->epphy);
 
 #ifdef CONFIG_USBDEV_DMAMEMORY
   usbdev_dma_free(buf);

